packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -561,18 +561,18 @@ async function callType3Webhook(action: 'post_published' | 'post_edited', url: s
  * Call the Type 3 webhook on post creation
  */
 getCollectionHooks("Posts").createAsync.add(async ({document}: CreateCallbackProperties<DbPost>) => {
-  if (!document.authorIsUnreviewed && !document.draft) {
-    const url = postGetPageUrl(document, true);
-    void callType3Webhook('post_published', url);
-  }
+  if (!isPostAllowedType3Audio(document)) return
+
+  const url = postGetPageUrl(document, true);
+  void callType3Webhook('post_published', url);
 });
 
 /**
  * Call the Type 3 webhook on post update. If the post is being undrafted this will count as being "published".
  * It will count as being "edited" if the post content has changed.
  */
 getCollectionHooks("Posts").updateAsync.add(async function updatedPostMaybeTriggerReview ({document, oldDocument}: UpdateCallbackProperties<DbPost>) {
-  if (document.draft || document.rejected) return
+  if (!isPostAllowedType3Audio(document)) return
```

Comment:
Just checking: in the case when we are editing a post to add a `podcastEpisodeId`, do we need to do something here or is it ok to just return?

Reply:
Yep it's fine, it will still hide the player correctly in `PostsPage`